std_msgs/String deploy_or_retrieve
std_msgs/String side
sensor_msgs/NavSatFix gps_pos